pinot-query-planner/src/main/java/org/apache/pinot/query/routing/ServerAddress.java

```diff
@@ -26,10 +26,16 @@ public class ServerAddress {
 
   private final String _hostname;
   private final int _port;
+  private final int _partition;
 
-  public ServerAddress(String hostname, int port) {
+  public ServerAddress(String hostname, int port, int partition) {
```

Comment:
Can we avoid calling this `partition`?

Partition has become quite overloaded now. One may confuse it with either query partition, data partition or Pinot's instance-partition.